scripts/test-uplink.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -ueo pipefail
+
+TMPDIR=$(mktemp -d -t tmp.XXXXXXXXXX)
+
+cleanup(){
+    rm -rf "$TMPDIR"
+    echo "cleaned up test successfully"
+}
+
+trap cleanup EXIT
+
+BUCKET=bucket-123
+SRC_DIR=$TMPDIR/source
+DST_DIR=$TMPDIR/dst
```

Comment:
Let's fix this thing in the test-aws script as well.

Reply:
Fixed

Reply:
did you push?